src/consensus/aft/raft.h

```diff
@@ -2409,23 +2542,36 @@ namespace aft
 
       LOG_INFO_FMT(
         "Becoming candidate {}: {}", state->my_node_id, state->current_view);
+      LOG_DEBUG_FMT("Configurations: {}", configuration_tracker.to_string());
 
       if (consensus_type != ConsensusType::BFT)
       {
         for (auto it = nodes.begin(); it != nodes.end(); ++it)
         {
+          if (it->first == state->my_node_id)
+            continue;
+
           channels->create_channel(
```

Comment:
Do we need to create channels to nodes that aren't eligible?

Reply:
Yes we do, during catchup we need to send them append entries and in BFT they also gossip their state to the other nodes.